### PR TITLE
Further consolidate Contributing guides

### PR DESCRIPTION
**Pull request by theletterf**
- Fixes https://github.com/open-telemetry/opentelemetry.io/issues/4652
- Preview: https://deploy-preview-4654--opentelemetry.netlify.app/docs/contributing/

### CODE REVIEW

File: README.md
Comment (chalin) [inline README.md:1]: What you added here as Development info should go into a separate page under the Contributing docs section, IMHO

Also, I think that this README should be changed to this:

```diff
diff --git a/README.md b/README.md
index cb27b13c..4a4337b6 100644
--- a/README.md
+++ b/README.md
@@ -27,9 +27,9 @@ to the registry][].
 
 ## Contributing
 
-We have curated some issues with the tags [help wanted][] and [good first
-issue][]. This should allow you to quickly find a place to contribute. See [CONTRIBUTING.md][]
-for more information.
+See the [Contributing](https://opentelemetry.io/docs/contributing) page on our docs.
+
+## Meetings
 
 We, the OTel Communications SIG, meet every two weeks on Monday at 10:00 PT.
 Check out the [OpenTelemetry community calendar][] for the Zoom link and any
@@ -38,6 +38,8 @@ updates to this schedule.
 Meeting notes are available as a public [Google doc][]. If you have trouble accessing
 the doc, get in touch in the `#otel-comms` channel on [Slack][].
 
+## Roles
+
 Here is a list of community roles with current and previous members:
 
 - Approvers: [@open-telemetry/docs-approvers][]
```

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,38 +3,17 @@
...
-
-### Contributor License Agreement
-
-Review the general
-[OpenTelemetry Contributor Guide](https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md),
-as it provides additional details, especially that you need to sign a
-Contributor License Agreement (CLA) before you can contribute.
+To learn how to contribute fixes and new content to this project, read the
+[Contributor's guide](https://opentelemetry.io/docs/contributing/), which
+includes a style guide and useful information on the review process.
```
Comment (chalin) [inline CONTRIBUTING.md:8]:
```suggestion
To learn how to contribute fixes and new content to this project, see
[Contributor](https://opentelemetry.io/docs/contributing/), which
includes a style guide and useful information on the review process.
```

**Review by chalin — COMMENTED** [review]
Thanks for this @theletterf. See inline comments.

**Review by svrnm — APPROVED** [review]

**Comment by theletterf** [comment]
@chalin Ready for a second look!

File: content/en/docs/contributing/requirements.md [hunk 2]
```diff
@@ -0,0 +1,662 @@
...
+Announcements will appear in the standard page order as determined using Hugo's
+[Regular pages](https://gohugo.io/methods/site/regularpages/) function. That is,
+the "lightest" announcements (by `weight`) will appear first; when weights are
+the same or unspecified, the most recent announcements (by `date`) will appear
+first, etc.
+
+So, if you want to force an announcement to the top, use a negative `weight` in
+the front matter.
+
+## Contribute to other repositories
```
[the remark this answers is not in the document]
Comment (theletterf) [reply content/en/docs/contributing/requirements.md:629]: Tempted to keep it. Let me try

**Review by chalin — APPROVED** [review]
🚀 🙏

**Pull request merged**